concepts/objects/about.md

```diff
@@ -0,0 +1,260 @@
+# About
+
+To use terminology from other languages, a JSON object is a "hash" or "map" or "dictionary".
+
+JSON defines an object as:
+
+> An object is an unordered set of **name**/**value** pairs.
+> An object begins with `{`left brace and ends with `}`right brace.
+> Each _name_ is followed by `:`colon and the _name/value_ pairs are separated by `,`comma.
+
+Further, the _name_ must be a string.
+Another word for _name_ is _key_.
+
+The _value_ can be of any JSON type.
+Different keys in the object can have values of different type, like this example.
+
+```json
+{
+  "name": "Jane",
+  "age": 42,
+  "pets": ["cat", "fish"],
+  "address": {"street": "123 Main St", "city": "Springfield"}
+}
+```
+
+Note that there **must not** be a comma following the _last_ key-value pair.
+
+## Creating objects
+
+Use braces to collect the name/value pairs.
+Even though the names must be strings, they do not need to be quoted if the names are "identifier-like" (composed of alphanumeric characters and underscore, and not started with a digit).
+
+```jq
+{name: "Jane", age: 42}
+```
+
+It is valid to use keys that are not "identifier-like".
+Just quote them.
+
+```jq
+{"first name": "Jane", "last name": "Lopez", age: 42}
+```
+
+If the name is the result of an expression, the expression **must** be in parentheses.
+
+```sh
+$ echo "Jane" | jq -Rc '{.: 42}'
+jq: error: syntax error, unexpected '.' (Unix shell quoting issues?) at <top-level>, line 1:
+{.: 42}
+jq: error: May need parentheses around object key expression at <top-level>, line 1:
+{.: 42}
+jq: 2 compile errors
+
+$ echo "Jane" | jq -Rc '{(.): 42}'
+{"Jane":42}
+```
+
+### Short-hand notations
+
+A couple of frequent cases for object construction have short-hand syntax.
+
+1. Re-using key-value pairs from an input object.
```

Comment:
I see consistent line endings prior to code blocks :smile:

Reply:
Lesson learned!